src/lib/server/trunk.c

```diff
@@ -361,6 +361,13 @@ static fr_table_num_indexed_bit_pos_t const fr_trunk_conn_trigger_names[] = {
 };
 static size_t fr_trunk_conn_trigger_names_len = NUM_ELEMENTS(fr_trunk_conn_trigger_names);
 
+static fr_table_num_ordered_t const fr_trunk_states[] = {
```

Comment:
You used the ordered table type but the states aren't lexicographically ordered.